AllReadyApp/Web-App/AllReady/Areas/Admin/ViewModels/Request/AddRequestCommentViewModel.cs

```diff
@@ -0,0 +1,12 @@
+namespace AllReady.Areas.Admin.ViewModels.Request
+{
+    public class AddRequestCommentViewModel
+    {
+        // possibly have a list of the comments which will be displayed below the dialog boxes
+
```

Comment:
Great idea @stevejgordon, would you mind adding a corresponding issue so that we can track this into the getter command for this VM?